index.bs

```diff
@@ -7119,9 +7120,11 @@ To <dfn>get the realm info</dfn> given |environment settings|:
 
     <dt>|global object| is a {{DedicatedWorkerGlobalScope}} object
     <dd>
+      1. Let |parent realm info| be the result of [=get the realm info=] given [=set up a
```

Comment:
I don't understand how this is supposed to work. We can't retrospectively pull out the argument that would have been passed to `set up a worker environment environment settings object` like this.

Reply:
This isn't retrospectively though because there is a statement saying _immediately prior to returning the settings
object_. This implies this is all part of the `set up a worker environment environment settings object` algorithm.

Reply:
Alternatively, in the `realmCreated` remote end event trigger, we can write after

> 1. Let |environment settings| be the newly created [=environment settings
   object=].
   
1. If the invoked algorithm is not [=set up a window environment settings object=], let |outside settings| be the [=outside settings=] parameter of the invoked algorithm.

Reply:
`outside settings` will always exist in other algorithms.

Reply:
Done.